migrations/v2.js

```diff
@@ -0,0 +1,181 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-assessment - v2.0.0 > v2.0.3', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.0.0', { name: 'adapt-contrib-assessment', version: '<2.0.3' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    return assessments.length;
+  });
```

Comment:
We could probably simplify this further to a single filter. 

Thoughts on changing assessments to assessmentArticles? 

```suggestion
  whereContent('adapt-contrib-assessment - where assessment', async content => {
    assessmentArticles = content.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
    return assessments.length;
  });
```

Reply:
Good call.